docs/source/guides/speed.rst

```diff
@@ -102,7 +89,7 @@ This by default comes with a performance hit, and can be disabled in most cases.
         strategy=DDPSpawnStrategy(find_unused_parameters=False),
     )
 
-When using DDP on a multi-node cluster, set NCCL parameters
+When using DDP on a Multi-Node Cluster, set NCCL Parameters
```

Comment:
https://capitalizemytitle.com/#capitalizationrules
https://www.grammarly.com/blog/capitalization-in-the-titles/

Using these rules would help us being consistent.